### DISCUSSION TITLE
How can we easily update our icons?

### DISCUSSION THREAD

**Discussion by soulcramer** [leboncoin/spark-android#250] [Decision making]
Currently we use the [VectorDrawbles xml](https://github.com/adevinta/spark-android/tree/main/spark-icons/src/main/res-iconography) that are from Brikke and then reference them through our [SparkIcon type](https://github.com/adevinta/spark-android/blob/main/spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt) but the process was done manually through Studio.

I think it would be simpler if we do like _Material_ and _Decathlon_, by converting a them to the ImageVector type which is compose specific by forking this [repo](https://github.com/DevSrSouza/svg-to-compose) to our needs 🤔

WDYT?

**Comment by SimonMarquis** [comment]
On the one hand I agree, but on the other hand they are already imported in the project, and migrating would require some work.
Do Material uses a public tool to generate those images we could use instead?
